indexer-core/src/main/java/org/apache/maven/index/ArtifactInfo.java

```diff
@@ -431,7 +429,8 @@ public Gav calculateGav()
     public String toString()
     {
         final StringBuilder result = new StringBuilder( getUinfo() );
-        if ( !Strings.isNullOrEmpty( getPackaging() ) )
+        String packaging = getPackaging();
+        if (packaging != null && !getPackaging().isEmpty())
```

Comment:
Don't call the method here, it could yield null.

Reply:
Oops my mistake.